src/main/java/gtPlusPlus/xmod/gregtech/loaders/recipe/RecipeLoader_GlueLine.java

```diff
@@ -224,28 +236,45 @@ private static void dehydratorRecipes() {
                 },
                 new int[]{10000, 10000, 10000},
                 300*20,
-                400);
+                10);
     }
 
     private static void distillationTowerRecipes() {
         GT_Values.RA.addDistillationTowerRecipe(
                 MISC_MATERIALS.CHLOROACETIC_MIXTURE.getFluidStack(1000),
                 new FluidStack[] {
-                        MISC_MATERIALS.CHLOROACETIC_ACID.getFluidStack(200),
-                        MISC_MATERIALS.DICHLOROACETIC_ACID.getFluidStack(400),
-                        MISC_MATERIALS.TRICHLOROACETIC_ACID.getFluidStack(400)
+                        MISC_MATERIALS.CHLOROACETIC_ACID.getFluidStack(100),
+                        MISC_MATERIALS.DICHLOROACETIC_ACID.getFluidStack(450),
+                        MISC_MATERIALS.TRICHLOROACETIC_ACID.getFluidStack(450)
                 },
                 null,
                 4 * 20,
-                MaterialUtils.getVoltageForTier(4));
+                MaterialUtils.getVoltageForTier(5));
+    }
+
+    private static void electrolyzerRecipes() {
+        GT_Values.RA.addElectrolyzerRecipe(
+                CI.getNumberedCircuit(1),
+                MISC_MATERIALS.CALCIUM_CHLORIDE.getDust(1),
```

Comment:
why 1 CaCl2 dust can be electrolyzed into 1 Ca dust and 2000mb Cl? you should keep all materials process consist with others. also there's already CaCl2 material ingame, why register duplicate material?

Reply:
I thought there was, but it must have been in a different mod that wasn't in my dev environment since I didn't see any when testing. If it already exists, I'll remove this recipe and new material and refer to that one instead. Thank you for letting me know.